### PR TITLE
Removing Remaining App Function References from Deployment Code.

### PR DESCRIPTION
**Pull request by nharper285**
## Summary of the Pull Request

_What is this about?_
Removing remaining `-net` references in deployment and agent code. 

## PR Checklist
* [ ] Applies to work item: #2676
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

### CODE REVIEW

File: src/deployment/azuredeploy.bicep [hunk 1]
```diff
@@ -192,21 +179,6 @@ resource roleAssignments 'Microsoft.Authorization/roleAssignments@2020-10-01-pre
...
-  }
-  dependsOn: [
-    eventGrid
-    keyVault
-    serverFarm
-    featureFlags
-  ]
-}]
-
 // try to make role assignments to deploy as late as possible in order to have principalId ready
```
Comment (mgreisen) [inline src/deployment/azuredeploy.bicep:182]: Did you mean to remove this comment?

Comment (nharper285) [reply src/deployment/azuredeploy.bicep:182]: I removed the entire roleAssignment resource and the comment that preceded it, yes.

**Review by mgreisen — APPROVED** [review]
:shipit:

File: src/deployment/deploy.py [hunk 2]
```diff
@@ -310,15 +308,13 @@ def get_instance_urls(self) -> List[str]:
                 "https://%s/%s" % (self.multi_tenant_domain, name)
                 for name in [
                     self.application_name,
-                    self.application_name + DOTNET_APPLICATION_SUFFIX,
                 ]
             ]
         else:
             return [
                 "https://%s.azurewebsites.net" % name
                 for name in [
```
Comment (AdamL-Microsoft) [inline src/deployment/deploy.py:316]:
```suggestion
        if self.multi_tenant_domain:
            return "https://%s/%s" % (self.multi_tenant_domain, self.application_name)
        else:
            return "https://%s.azurewebsites.net" % self.application_name
```

Comment (AdamL-Microsoft) [reply src/deployment/deploy.py:316]: I mixed up the line number here, but these two blocks can be condensed into just the returns

File: src/deployment/deploy.py [hunk 3]
```diff
@@ -333,15 +329,13 @@ def get_identifier_urls(self) -> List[str]:
                 "api://%s/%s" % (self.multi_tenant_domain, name)
                 for name in [
                     self.application_name,
```
Comment (AdamL-Microsoft) [inline src/deployment/deploy.py:331]: same condensed returns as above apply here too

**Review by AdamL-Microsoft — APPROVED** [review]

**Comment by mgreisen** [comment]
closes #2676

**Pull request merged**